### ISSUE TITLE
V160 Crash "bds.exe" when the IDE is closed and various findings

### ISSUE THREAD

**Issue by Rollo62b** [DelphiWorlds/Codex#33]
Hi Dave,

are there any known issues ?
I have tried Codex 1.6.0 and 1.6.0_Beta, together with the Mosco 1.4.1 and found some issues.
Using RadStudio 11.3 ( only last Patch 1 1.0 not yet installed )

1. I couldn't really find which one is most current, the codex or the Codex beta. Both have same file dates.
    Which one is the most current, I assume the Beta, right ?
2. Codex Beta Setup crashes when installing, but seems to install anyway
3. Codex Setup doesn't crash and installs as well 
4. Both seems to run basically, but only when I close the IDE, both ( or the IDE ) crashes with AV ( open / close the IDE is enough )
   To provoke this issue.   
   ![grafik](https://github.com/DelphiWorlds/Codex/assets/4697968/be3c7a2e-83f0-424f-b46b-11c151b5b2f4)
   Nevertheless, seems to work when running.
   After uninstalling Codex, the IDE works normal again.
   
    **Edit:**
   It seems that several processes of the IDE keeps open, when that happes several times.
   After such crash, the IDE needs to be killed manually.
5. In the Codex beta, there is the option to unpack AAB, but it's not in normal Codex. Was this added or removed ?
    This unpack function don't work for me, I could enter all 3 values AAB, KeyStore file and password, but the OK button still was greyed.
6. Trying the option to connect to the Mosco 1.4.1 server, I got a message 12152 when pressing "Test".
    "Error receiving data: (12152) The server delivered an invalid or unknown message.
    The Mosco was running and could be tested in the IDE ConnectionProfileManager wit the same IP adress and port.
    
Maybe that helps to fix some issues.

**Comment by DelphiWorlds** [comment]
Apologies.. I've only just noticed this issue was here!

1. Codex 1.6.0 release version is the most current
2. Not sure why - seems to install fine here?
3. OK
4. Does it happen all the time? 
5. It was removed from the release version. If it's something useful to you, let me know
6. Does it happen all the time?

Note that I've stopped all work on v1.6.x, and have been working on v2.0.0, for which a beta is planned soon as well as release of the source. "Official" release of v2.0.0 is planned for early November.
